mlflow/pmdarima.py

```diff
@@ -0,0 +1,241 @@
+        _save_example(mlflow_model, input_example, path)
+
+    model_data_path = os.path.join(path, _MODEL_BINARY_FILE_NAME)
+    _save_model(pmdarima_model, model_data_path)
+
+    model_bin_kwargs = {_MODEL_BINARY_KEY: _MODEL_BINARY_FILE_NAME}
+    pyfunc.add_to_model(
+        mlflow_model, loader_module="mlflow.pmdarima", env=_CONDA_ENV_FILE_NAME, **model_bin_kwargs
+    )
+    flavor_conf = {
+        _MODEL_TYPE_KEY: pmdarima_model.__class__.__name__,
+        **model_bin_kwargs,
+    }
+    mlflow_model.add_flavor(FLAVOR_NAME, pmdarima_version=pmdarima.__version__, **flavor_conf)
+    mlflow_model.save(os.path.join(path, MLMODEL_FILE_NAME))
+
+    if conda_env is None:
+        if pip_requirements is None:
+            default_reqs = get_default_pip_requirements()
+            inferred_reqs = mlflow.models.infer_pip_requirements(
+                path, FLAVOR_NAME, fallback=default_reqs
+            )
+            default_reqs = sorted(set(inferred_reqs).union(default_reqs))
+        else:
+            default_reqs = None
+        conda_env, pip_requirements, pip_constraints = _process_pip_requirements(
+            default_reqs, pip_requirements, extra_pip_requirements
+        )
+    else:
+        conda_env, pip_requirements, pip_constraints = _process_conda_env(conda_env)
+
+    with open(os.path.join(path, _CONDA_ENV_FILE_NAME), "w") as f:
+        yaml.safe_dump(conda_env, stream=f, default_flow_style=False)
+
+    if pip_constraints:
+        write_to(os.path.join(path, _CONSTRAINTS_FILE_NAME), "\n".join(pip_constraints))
+
+    write_to(os.path.join(path, _REQUIREMENTS_FILE_NAME), "\n".join(pip_requirements))
+
+
+@format_docstring(LOG_MODEL_PARAM_DOCS.format(package_name=FLAVOR_NAME))
+def log_model(
+    pmdarima_model,
+    artifact_path,
+    conda_env=None,
+    registered_model_name=None,
+    signature: ModelSignature = None,
+    input_example: ModelInputExample = None,
+    await_registration_for=DEFAULT_AWAIT_MAX_SLEEP_SECONDS,
+    pip_requirements=None,
+    extra_pip_requirements=None,
+    **kwargs,
+):
+    """
+    :param pmdarima_model:
+    :param artifact_path:
+    :param conda_env:
+    :param registered_model_name:
+    :param signature:
+    :param input_example:
+    :param await_registration_for:
+    :param pip_requirements:
+    :param extra_pip_requirements:
+    :param kwargs:
+
+    :return:
+    """
+
+    return Model.log(
+        artifact_path=artifact_path,
+        flavor=mlflow.pmdarima,
+        registered_model_name=registered_model_name,
+        pmdarima_model=pmdarima_model,
+        conda_env=conda_env,
+        signature=signature,
+        input_example=input_example,
+        await_registration_for=await_registration_for,
+        pip_requirements=pip_requirements,
+        extra_pip_requirements=extra_pip_requirements,
+        **kwargs,
+    )
+
+
+def load_model(model_uri, dst_path=None):
+    """
+    :param model_uri:
+    :param dst_path:
+    """
```

Comment:
Can we complete these parameter docstrings and add a description of the overall method?